cmd/api/src/api/v2/anonymize.go

```
@@ -0,0 +1,342 @@
+	common.Email,
+	common.OperatingSystem,
+	common.Title,
+}
+
+// generatePseudonym creates a random hex pseudonym with the given prefix.
+// Uses 8 bytes (64-bit) of entropy for the suffix.
+func generatePseudonym(prefix string, seen map[string]struct{}) string {
+	for attempts := 0; attempts < 10; attempts++ {
+		b := make([]byte, 8)
+		if _, err := rand.Read(b); err != nil {
+			slog.Warn("generatePseudonym: crypto/rand.Read failed, using fallback",
```

golangci-lint (2.11.4)

[major] message should be capitalized

(sloglint)